### DISCUSSION TITLE
Savings Session - not charging ahead of the session

### DISCUSSION THREAD

**Discussion by gmarchment** [springfall2008/batpred#809] [General]
I'm not sure why the battery isn't Charging to then Discharge during the saving session later today.  Any ideas?

Also - I've noticed a "Predbat Active" toggle in the switches - what does this do?

<img width="468" alt="image" src="https://github.com/springfall2008/batpred/assets/151647330/a7ff9dbf-de0e-4170-8e98-89e69b2456af">

**Comment by gmarchment** [comment]
Weird - for some reason Predbat had dropped on to Monitor mode.  When I put it back to Control Charge and Discharge, I got this;

<img width="577" alt="image" src="https://github.com/springfall2008/batpred/assets/151647330/40bbbae8-39e1-4282-b43c-61c67b2cde6b">

Why wouldn't it charge to ~80% as I know it can probably push out around 75% of the battery in a 30min window.

**Reply by iainfogg** [reply]
@gmarchment that doesn't sound right to me, what's your setup?

A battery that can discharge 75% of its capacity in 30 mins isn't something I've come across.

Mine can discharge just under 2 kWh in 30 mins.

**Comment by gcoan** [comment]
Saving sessions finished for this year so this looks like a stale issue
